tests/failpoints/cases/test_conf_change.rs

```diff
@@ -158,8 +158,6 @@ fn test_tick_after_destroy() {
     let engine_3 = cluster.get_engine(3);
     must_get_equal(&engine_3, b"k1", b"v1");
 
-    let tick_fp = "on_raft_log_gc_tick_1";
-    fail::cfg(tick_fp, "pause").unwrap();
```

Comment:
Change this failpoint to function `on_raft_gc_log_tick` and use `return` not `pause` then set `raft_log_gc_tick_interval` to 50ms in this test.